test_scripts/Polices/Policy_Table_Update/007_ATF_P_TC_OnStatusUpdate_UPDATE_NEEDED_new_PTU_request.lua

```diff
@@ -59,8 +60,10 @@ end
 --[[ Test ]]
 commonFunctions:newTestCasesGroup("Test")
 function Test:TestStep_OnStatusUpdate_UPDATE_NEEDED_new_PTU_request()
-  self.hmiConnection:SendNotification("SDL.OnPolicyUpdate", {} )
+  local RequestIdUpdateSDL = self.hmiConnection:SendRequest("SDL.UpdateSDL")
+  EXPECT_HMIRESPONSE(RequestIdUpdateSDL,{result = {code = 0, method = "SDL.UpdateSDL", result = "UP_TO_DATE" }})
```

Comment:
@dboltovskyi, as user request new PTU result of SDL.UpdateSDL should be "UPDATE_NEEDED".